src/auth/users.respository.ts

```diff
@@ -41,4 +43,35 @@ export class UsersRepository extends Repository<User> {
     const user = this.findOne({ where: { id: idUser } });
     return user;
   }
+
+  async updateUserRole(userId: string, role: UserRole): Promise<User> {
+    const query = this.createQueryBuilder();
+    const toUpperCase: string = role.toUpperCase();
+    const getRole: UserRole = UserRole[toUpperCase];
+    try {
+      // const updated = await this.update(userId, { role: getRole });
+      const res = await query
+        .update(User)
+        .set({ role: getRole })
+        .where('id =:userId', { userId })
+        .execute();
+
+      if (res.affected === 1) {
+        const user = this.getUser(userId);
+        return user;
+      } else {
+        throw new NotFoundException(`The user doesn't exist`);
+      }
+    } catch (error) {
+      throw new ConflictException('check the enum input');
+    }
+  }
+
+  async getRolesForUser(): Promise<object> {
+    const roleForUsers = Object.keys(UserRole).map((key) => ({
+      Name: key,
+      Role: UserRole[key],
```

Comment:
The convention applied so far was for all keys to start with a lowercase letter(camel case).
```
{
     name: key,
     role: UserRole[key],
}
```

Reply:
u mean to return like this ? 
```
[
    {
        "role": "user"
    },
    {
        "role": "admin"
    },
    {
        "role": "moderator"
    }
]
```